docs/QSG/tour.md

```diff
@@ -242,11 +242,13 @@ bring-up DAOS servers and clients.
 	$ dmg pool list
 	Pool UUID								Svc Replicas
 	---------								------------
-	5f362dc2-6154-44c7-8348-9de6f0a3d5d1	0
+	bf9a1b61-e3ba-4dca-a4ba-f35776edc822	0
+	13aaa333-1533-41cb-85d8-8a22728cfbad	0
+	f73eefa5-937d-4f0d-8934-4b3192dba4e4	0
```

Comment:
The pool list output is significantly different on 2.0. This needs to be updated.

Reply:
Will update, thanks.